### PR TITLE
[Text Analytics] Add release notes for November

### PR DESCRIPTION
**Pull request by deyaaeldeen**
This adds release notes for text analytics November beta release.

### CODE REVIEW

File: releases/2020-11/js.md [hunk 1]
```diff
@@ -222,6 +224,17 @@ In addition to the above form recognition features, there is also a new set of m
...
 
+### Text Analytics
+
+#### @azure/ai-text-analytics [Changelog](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/textanalytics/ai-text-analytics/CHANGELOG.md)
+
+This beta package targets Azure Text Analytics API version  `3.1-preview.3`, and introduces support for its new text analytics features, including:
+
+##### New Features on @azure/ai-text-analytics@5.1.0-beta.3
+
+- Support for the new API for batch tasks processing through the `beginAnalyze` method. Current list of supported tasks is Named Entity Recognition, Personally identifiable Information, and Key Phrase Extraction.
```
Comment (sadasant) [inline releases/2020-11/js.md:235]:
```suggestion
- Support for the new API for batch tasks processing through the `beginAnalyze` method. The current supported tasks are: Named Entity Recognition, Personally identifiable Information, and Key Phrase Extraction.
```

**Review by sadasant — APPROVED** [review]
Thank you! Approved!

File: releases/2020-11/js.md [hunk 2]
```diff
@@ -222,6 +224,17 @@ In addition to the above form recognition features, there is also a new set of m
...
+### Text Analytics
+
+#### @azure/ai-text-analytics [Changelog](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/textanalytics/ai-text-analytics/CHANGELOG.md)
+
+This beta package targets Azure Text Analytics API version  `3.1-preview.3`, and introduces support for its new text analytics features, including:
+
+##### New Features on @azure/ai-text-analytics@5.1.0-beta.3
+
+- Support for the new API for batch tasks processing through the `beginAnalyze` method. The currently supported tasks are Named Entity Recognition, Personally identifiable Information, and Key Phrase Extraction.
+- Support for the recognition of healthcare entities API with the introduction of the `beginAnalyzeHealthcare` method. Since the Health API is currently only available in a gated preview, you need to have your subscription on the service's allow list. Also, note that since this is a gated preview, AAD is not supported. For more information, see (the Text Analytics for Health documentation)[https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/how-tos/text-analytics-for-health?tabs=ner#request-access-to-the-public-preview].
```
Comment (maririos) [inline releases/2020-11/js.md:236]: late to the party, but if possible remove the locale `en-us` from the link.
Also, is it intentional that the link is not hyperlink ?

Comment (deyaaeldeen) [reply releases/2020-11/js.md:236]: @maririos good catches! Opened https://github.com/Azure/azure-sdk/pull/2080

**Pull request merged**